From 99caaae2b71b6bc88fdee1eecfa5cc1feb250eba Mon Sep 17 00:00:00 2001
From: Zakariyya Mughal <zaki.mughal@gmail.com>
Date: Wed, 28 Feb 2018 10:51:33 -0500
Subject: [PATCH 2/4] Add include guard on third-party library source files

---
 willuslib/ocrtess.c | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/willuslib/ocrtess.c b/willuslib/ocrtess.c
index e1366e1..377ad64 100644
--- a/willuslib/ocrtess.c
+++ b/willuslib/ocrtess.c
@@ -24,9 +24,12 @@
 #include <stdlib.h>
 #include <string.h>
 #include <ctype.h>
+#include "willus.h"
+
+#ifdef HAVE_TESSERACT_LIB
+
 #include <leptonica.h>
 #include <tesseract.h>
-#include "willus.h"
 
 
 static int ocrtess_lstm_file(char *filename);
@@ -589,3 +592,5 @@ static void endian_flip(char *x,int n)
         x[2]=c;
         }
     }
+
+#endif /* HAVE_TESSERACT_LIB  */
-- 
2.20.1

