From 65694d71f443855dbfc1a63c9915713dd234776c Mon Sep 17 00:00:00 2001
From: Zakariyya Mughal <zaki.mughal@gmail.com>
Date: Wed, 28 Feb 2018 10:52:40 -0500
Subject: [PATCH 3/6] Dewarp requires Leptonica

---
 k2pdfoptlib/k2master.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/k2pdfoptlib/k2master.c b/k2pdfoptlib/k2master.c
index 457b3ad..9b7a12f 100644
--- a/k2pdfoptlib/k2master.c
+++ b/k2pdfoptlib/k2master.c
@@ -224,6 +224,7 @@ printf("masterinfo->landscape=%d\n",masterinfo->landscape);
         }
     if (k2settings->autocrop)
         bmp_autocrop2(srcgrey,masterinfo->autocrop_margins,(double)k2settings->autocrop/1000.);
+#ifdef HAVE_LEPTONICA_LIB
     if (k2settings->dewarp && !k2settings->use_crop_boxes)
         {
         WILLUSBITMAP _dwbmp,*dwbmp;
@@ -261,6 +262,7 @@ printf("masterinfo->landscape=%d\n",masterinfo->landscape);
         if (k2settings->autocrop)
             bmp_autocrop2(srcgrey,masterinfo->autocrop_margins,(double)k2settings->autocrop/1000.);
         }
+#endif /* HAVE_LEPTONICA_LIB */
 #if (WILLUSDEBUGX & 0x20000)
 printf("k2settings->srccropmargins->units[0]=%d\n",k2settings->srccropmargins.units[0]);
 #endif
-- 
2.16.1

